webapp/src/main/java/com/box/l10n/mojito/service/ai/RepositoryLocaleAIPromptRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.box.l10n.mojito.service.ai;
+
+import com.box.l10n.mojito.entity.RepositoryLocaleAIPrompt;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface RepositoryLocaleAIPromptRepository
+    extends JpaRepository<RepositoryLocaleAIPrompt, Long> {
+
+  @Query(
+      "SELECT count(rlap.id) FROM RepositoryLocaleAIPrompt rlap "
+          + "JOIN AIPrompt aip ON rlap.aiPrompt.id = aip.id "
+          + "JOIN AIPromptType aipt ON aip.promptType.id = aipt.id "
+          + "WHERE rlap.repository.id = :repositoryId AND rlap.disabled = false AND aip.deleted = false AND aipt.name = 'TRANSLATION'")
```

Comment:
Question for the two methods in this file: I'm wondering if this should be `findCountOfActiveRepositoryPromptsOfType` - where the type (TRANSLATION, etc.) is passed in as a param from an enum, vs. hardcoded in the SQL (and needing multiple similar queries later).

We can always refactor this whenever we add new names/types later though - so not a critical.

Reply:
Small update to do now so I'll include it in this PR